src/backend/app/tasks/task_routes.py

```diff
@@ -19,6 +19,116 @@
 )
 
 
+@router.post("/{task_id}")
+async def create_task_comment(
```

Comment:
I dont think we need seperate endpoint for task comment. Can we use the task events update with state as `comment` and messgae with the comment message.